src/installer/Directory.Build.props

```diff
@@ -61,8 +61,8 @@
   -->
 
   <PropertyGroup>
-    <OSGroup Condition="'$(OS)'=='Unix' AND Exists('/Applications')">OSX</OSGroup>
-    <OSGroup Condition="'$(OSGroup)' == ''">$(OS)</OSGroup>
+    <BuildOS Condition="'$(OS)'=='Unix' AND Exists('/Applications')">OSX</BuildOS>
+    <BuildOS Condition="'$(BuildOS)' == ''">$(OS)</BuildOS>
```

Comment:
Shouldn't we move the BuildOS calculation out of installer and also remove the duplication from other parts in the repo into eng/Configurations.props?